src/app/views/change-password/changePasswordRequest.jsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const propTypes = {
+    validationErrors: PropTypes.shape({
+        hading: PropTypes.string,
+        body: PropTypes.oneOfType([PropTypes.string, PropTypes.element])
+    }),
+    onSubmit: PropTypes.func,
+    emailInput: PropTypes.shape({
+        id: PropTypes.string,
+        label: PropTypes.string,
+        type: PropTypes.string,
+        onChange: PropTypes.func,
+        error: PropTypes.string
+    })
+};
+
+const ChangePasswordRequest = props => {
+    // TODO this page is to be done in an upcoming task
+    return (
+        <div className="grid grid--justify-center">
+            <div className="grid__col-3">
+                <h1>We are on change password request</h1>
```

Comment:
As this will be deployed to prod (although behind a feature flag) we need to make sure on the off chance someone stumbles upon the page before we change this heading that they don't see irrelevant comments. We should probably change this h1 to say something like "Request password change" as "We are on change password request" is quite informal.